java/client/src/main/java/glide/connectors/handlers/ReadHandler.java

```diff
@@ -31,6 +31,7 @@ public void channelRead(@NonNull ChannelHandlerContext ctx, @NonNull Object msg)
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
         // TODO: log thru logger
         System.out.printf("=== exceptionCaught %s %s %n", ctx, cause);
+        cause.printStackTrace();
```

Comment:
remove?

Reply:
I think we need to log it through logger too.